robot_skills/src/robot_skills/arms.py

```diff
@@ -140,7 +144,7 @@ def cb_hardware_status(self, msg):
             else:
                 self._operational = True
 
-    def send_goal(self, px, py, pz, roll, pitch, yaw,
+    def send_goal(self, frame,
```

Comment:
Good point. In what frame is the self.offset defined? Baselink I assume? 

The FrameStamped has to be converted to that frame first before the offset can be applied